Viktoriia/README.md

```diff
@@ -0,0 +1,4 @@
+1. I will make the graphic part of the program.(FieldImage function)
+2. To implement the alternation of players, I will use recursion.(Cross function, Zero function)
```

Comment:
Using recursion here is not the best choice. Recursion is quite complex to manage and also takes a lot of space.

Better way would be to use `while` loop which will run until game is not finished. Inside while loop current player will be changed and the rest of the steps will be same for both players. Here is pseudocode for this approach: 
```c
while(!game_is_finished(gaming_field)) {
    change_active_player()
    let_a_player_to_make_a_move()
    render_gaming_board()
}
```